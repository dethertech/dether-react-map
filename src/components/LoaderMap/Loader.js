/**
 * Dependencies
 */
import React from 'react'

/**
 * CSS
 */
import './Loader.style.scss'

/**
 * Loader component
 */
const LoaderMap = () => (
  <div id="preloader" style={{ zIndex: 1099 }}>
    <div id="loader" />
  </div>
)

export default LoaderMap
