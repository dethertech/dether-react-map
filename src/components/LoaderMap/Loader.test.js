/* global describe beforeEach it expect test */

/**
 * Dependencies
 */
import React from 'react'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'

/**
 * Components
 */
import Loader from './index'

/**
 * TESTS
 */
test('Loader is rendering properly', () => {
  const component = renderer.create(<Loader />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})

describe('Loader', () => {
  let wrapper = null

  beforeEach(() => {
    wrapper = shallow(<Loader />)
  })

  it('should render the loader component', () => {
    expect(wrapper.length).toEqual(1)
  })
})
