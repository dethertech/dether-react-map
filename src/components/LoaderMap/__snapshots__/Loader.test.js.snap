// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader is rendering properly 1`] = `
<div
  className="app-start-loading"
>
  <div
    id="preloader"
  >
    <div
      id="loader"
    />
  </div>
</div>
`;
